.travis.yml

```diff
@@ -4,7 +4,143 @@ jdk:
+- knime_3.7.2/knime -application org.eclipse.equinox.p2.director -lir
+
+- |
+  for WF in $WF_FILES
+  do
+ 
+  KNIME_OUT=$(knime_3.7.2/knime -nosplash -reset -nosave -application org.knime.product.KNIME_BATCH_APPLICATION -workflowFile="$WF" --launcher.suppressErrors)
+  if [[ "$KNIME_OUT" != *"$KNIME_SUCCESS"* ]]; then  echo "Workflow failed";exit 1; else echo "3.7.2 WORKFLOW SUCCESSFUL ON UPDATE"; fi
+  rm outfile.table
+
+  done
+
+
+- echo "CLEANING KNIME"
+- rm -Rf knime_3.7.2
+
+# -------------------------------------------DO THE SAME FOR KNIME 4.1.2 -------------------------------------------------------------------
+
+- KNIME_FILE="knime_4.1.2.linux.gtk.x86_64.tar.gz"
+- wget -v "http://download.knime.org/analytics-platform/linux/$KNIME_FILE"
+- tar -xzf $KNIME_FILE
+- rm $KNIME_FILE
+- KNIME41="https://update.knime.com/analytics-platform/4.1"
+
+
+- echo "================== TESTING THE FRESH INSTALL OF FSK-LAB of KNIME 4.1.2 ==============================================================="
+
+- echo "INSTALL NEW FSK-LAB INTO FRESH KNIME"
+- knime_4.1.2/knime -application org.eclipse.equinox.p2.director -repository "$KNIME41,$NEW_FSK" -installIU de.bund.bfr.knime.fsklab.feature.feature.group,org.knime.features.testing.application.feature.group
+- knime_4.1.2/knime -application org.eclipse.equinox.p2.director -lir
+-
+- |
+  for WF in $WF_FILES
+  do
+ 
+  knime_4.1.2/knime -nosplash -reset -nosave -application org.knime.product.KNIME_BATCH_APPLICATION -workflowFile="$WF" --launcher.suppressErrors
+  OUTFILE=outfile.table
+  if test -f "$OUTFILE"; then echo "4.1.2 WORKFLOW SUCCESSFUL ON FRESH INSTALL"; else echo "WORKFLOW NOT EXECUTED";exit 1; fi
+  rm outfile.table
+
+  done
+
+
+- echo "REMOVE NEW FSK TO TEST UPDATE FROM OLDER VERSION"
+- knime_4.1.2/knime -application org.eclipse.equinox.p2.director -uninstallIU de.bund.bfr.knime.fsklab.feature.feature.group
+- knime_4.1.2/knime -application org.eclipse.equinox.p2.director -lir
+-
+-
+- echo "================== TESTING THE UPDATE FROM OLD FSK-LAB ==============================================================================="
+-
+- echo "INSTALL OLD FSK-LAB TEMPORARILY"
+- knime_4.1.2/knime -application org.eclipse.equinox.p2.director -repository "$KNIME41,$OLD_FSK" -installIU de.bund.bfr.knime.fsklab.feature.feature.group
+- knime_4.1.2/knime -application org.eclipse.equinox.p2.director -lir
+- echo "REMOVING OLD FSK-LAB"
+- knime_4.1.2/knime -application org.eclipse.equinox.p2.director -uninstallIU de.bund.bfr.knime.fsklab.feature.feature.group
+- knime_4.1.2/knime -application org.eclipse.equinox.p2.director -lir
+- echo "REINSTALLING NEW FSK-LAB"
+- knime_4.1.2/knime -application org.eclipse.equinox.p2.director -repository "$KNIME41,$NEW_FSK" -installIU de.bund.bfr.knime.fsklab.feature.feature.group
+- knime_4.1.2/knime -application org.eclipse.equinox.p2.director -lir

```

Comment:
These commands listing the installed plugins are not needed and produce noise here. They are only used in manual tests. They should not be used in a "batch" mode.